canonical-websites/insights.ubuntu.com
# Rename to blog

## Done

Update logo, title, open graph images and meta tags

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- Make sure the logo says blog and not insights
- Make sure the page title is blog and not insights
